app/controllers/v0/preneeds/burial_forms_controller.rb

```diff
@@ -1,22 +1,41 @@
 # frozen_string_literal: true
+require 'sentry_logging'
+
 module V0
   module Preneeds
     class BurialFormsController < PreneedsController
-      FORM = '40-10007'
+      include SentryLogging
 
-      def new
-      end
+      FORM = '40-10007'
 
       def create
         form = ::Preneeds::BurialForm.new(burial_form_params)
         validate!(form)
 
         resource = client.receive_pre_need_application(form.as_eoas)
+        log_submission(resource)
+
         render json: resource, serializer: ReceiveApplicationSerializer
       end
 
       private
 
+      def log_submission(resource)
+        record = ::Preneeds::PreneedSubmission.create(
+          tracking_number: resource.tracking_number,
+          application_uuid: resource.application_uuid,
+          return_description: resource.return_description,
+          return_code: resource.return_code
+        )
+
+        unless record.persisted?
```

Comment:
maybe change this to "if record.errors.any?" NITPICK, but more consistent with elsewhere.

Reply:
If it's really critical that it gets persisted and you're worried about ActiveRecord connection failures and stuff like that, then you'd probably need to do your PreneedSubmission.create in a worker that has retry enabled.

Reply:
If there is an ActiveRecord connection error, it will result in a 500 error to the user probably in which case it would get logged automatically provided you set the context for Sentry, FYI. Not sure your .persisted? will ever get triggered because the active record failure bubbles up above.